client/logmon/logmon.go

```diff
@@ -198,13 +198,16 @@ func (l *logRotatorWrapper) isRunning() bool {
 
 // newLogRotatorWrapper takes a rotator and returns a wrapper that has the
 // processOutWriter to attach to the stdout or stderr of a process.
-func newLogRotatorWrapper(path string, logger hclog.Logger, rotator *logging.FileRotator) (*logRotatorWrapper, error) {
+func newLogRotatorWrapper(path string, logger hclog.Logger, rotator io.WriteCloser) (*logRotatorWrapper, error) {
 	logger.Info("opening fifo", "path", path)
 
 	var openFn func() (io.ReadCloser, error)
 	var err error
 
-	if _, ferr := os.Stat(path); os.IsNotExist(ferr) {
+	//FIXME Revert #5990 and check os.IsNotExist once Go >= 1.12 is the
+	//      release compiler.
```

Comment:
Fix comment formatting?

Reply:
Ha, I have some weird old habit of indenting wrapped lines past the prefix. Fixed!